### ISSUE TITLE
Reordering new Vizy blocks that contain a static Super Table field causes new rows to be added to the Super Table

### ISSUE THREAD

**Issue by ellukesmith** [verbb/vizy#47]
**Description**
Reordering Vizy blocks that contain static Super Table fields will cause the Super Table field to add a new row, even though it's a static Super Table and should only contain a single row. This only impacts newly added Vizy blocks that haven't been saved against the entry yet. If the entry has been saved, fields can be re-ordered as expected.

**Steps to reproduce**
1. Create a new Super Table field.
2. Add the Super Table field to a Vizy block.
3. In the Vizy WYSIWYG editor, add two of the static Super Table blocks.
4. Without saving the entry, change the order of the blocks.

**Additional info**
- Plugin version: 1.0.2
- Craft version: Craft Pro 3.6.16
- Multi-site: Yes

**Additional context**
PHP Version: 7.4.16

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.5](https://github.com/verbb/vizy/releases/tag/1.0.5)
